ticdc/integrate-confluent-using-ticdc.md

```diff
@@ -1,130 +1,264 @@
 ---
-title: Quick Start Guide on Integrating TiDB with Confluent Platform
-summary: Learn how to stream TiDB data to the Confluent Platform using TiCDC.
+title: Integrate Data with Confluent Cloud
+summary: Learn how to stream TiDB data to the Confluent Cloud using TiCDC, and how to replicate incremental data to ksqlDB, Snowflake, and SQL Server.
 ---
 
-# Quick Start Guide on Integrating TiDB with Confluent Platform
+# Integrate Data with Confluent Cloud
 
-This document introduces how to integrate TiDB to Confluent Platform using [TiCDC](/ticdc/ticdc-overview.md).
+Confluent is a Apache Kafka-compatible streaming data platform that provides strong integration capabilities. On this platform, you can access, store, and manage non-stop real-time streaming data.
```

Comment:
```suggestion
Confluent is an Apache Kafka-compatible streaming data platform that provides strong integration capabilities. On this platform, you can access, store, and manage non-stop real-time streaming data.
```

Reply:
Fixed in [3fa1b2c](https://github.com/pingcap/docs/pull/9866/commits/3fa1b2c665c117963cd01ccdc9574a30475dada0)